build/index.js

```diff
@@ -36,6 +36,9 @@ let getCurrencyObject = function (currency) {
         }
     }
 };
+function isAmountObject(amount) {
+    return lodash_1.isObject(amount);
```

Comment:
It would be nice if Fowler's money pattern could be implemented in "clean" Typescript without using lodash at all.

Reply:
Also would love that

Reply:
Indeed, I might give it a try in the next few weeks.